web/cobrands/bathnes/assets.js

```diff
@@ -234,5 +234,52 @@ fixmystreet.assets.add(fixmystreet.maps.banes_defaults, {
     attribution: " © Crown Copyright. All rights reserved. 1000233344"
 });
 
+// List of categories which are Curo Group's responsibility
+var curo_categories = [
+    'Allotment issue',
+    'Overgrown public spaces & parks',
+    'Play area safety issue',
+    'Trees and woodland',
+    'Dead animals',
+    'Dog fouling',
+    'Excessive or dangerous littering',
+    'Household bins left out early or after collection day',
+    'Needles'
+];
+
+fixmystreet.assets.add(fixmystreet.maps.banes_defaults, {
+    http_options: {
+        params: {
+            TYPENAME: "Curo_Land_Registry"
+        }
+    },
+    asset_type: 'area',
+    stylemap: fixmystreet.assets.stylemap_invisible,
+    non_interactive: true,
+    always_visible: true,
+    all_categories: true, // Not really, but we deal with that in the found action handler.
+    road: true,
+    no_asset_msg_id: '#js-housing-association-restriction',
+    actions: {
+        found: function(layer) {
+            var category = $('select#form_category').val();
+            if (curo_categories.indexOf(category) === -1) {
+                fixmystreet.message_controller.hide_responsibility_message();
+                return;
+            }
+
+            var email = 'estates@curo-group.co.uk';
+            if (category === 'Household bins left out early or after collection day') {
+                email = 'tennancycompliance&support@curo-group.co.uk';
+            }
+            var email_string = $(layer.fixmystreet.no_asset_msg_id).find('.js-email');
+            if (email_string) {
+                email_string.html('<a href="mailto:' + email + '">' + email + '</a>');
+            }
+            fixmystreet.message_controller.show_responsibility_message(layer);
+        },
+        not_found: fixmystreet.message_controller.hide_responsibility_message,
```

Comment:
I would prefer this to use the existing road_found/road_not_found, given my current work, which I think should be possible. `not_found` here would match to message_controller.road_found fine, and `found` would match to road_found if not a matching category, and could set asset_item before calling road_not_found (much as the streets asset layer in the same file already does). Then no need for the new message_controller functions.
It will need to behave differently in multi-page anyway, I'm hoping keeping the same mechanism will mean less to have to change and it might carry across fine. And then any tweaks can be made.

Reply:
There were a couple of reasons that I didn't use `road_found`/`road_not_found`.

Firstly, I want to be able to change the text of the `<strong>` tag in `#js-roads-responsibility` to have different text, rather than use the "Not maintained by Bath &amp; North East Somerset Council" that's there at the moment. But `#js-roads-res

The text for `asset_item` says `a <b class="asset-…`, which isn't appropriate for an email address.

Looking at the multi-page forms stuff I can see that making any changes to that stuff right now isn't really a good idea, so I've tried to work around the above issues in the found: handler. Bit fragile, but I think better than meddling with the core code.

Reply:
On the strong, your sentence doesn't end; but I think in the BANES template you could move that strong inside the inner div and then it would work as you want, you wouldn't need the special show/hide then?

Ah yes, the "a..." makes it more annoying, doesn't it.Your solution to change the email looks good, thanks for adapting the code!